### ISSUE TITLE
where are the default log files are saved? can the path be changed?

### ISSUE THREAD

**Issue by mahermeg17** [savio-zc/Android-Logger#1]
Hi,

where are the default log files are saved? can the file name and path be changed?

thank you.

**Comment by mahermeg17** [comment]
OK. I found the path "/storage/sdcard/AndroidLogger/com.xxx.xxx"
can I change it ?

**Comment by savio-zc** [comment]
No, you can not change the path. If you really want to do so, fork the project and change the code in LogManagerConfig.java.
